src/cmd/purge/scripts/setRetention.ps1

```diff
@@ -34,8 +34,7 @@ Get-Mailbox | ForEach-Object {
         -Force
 }
 
-Write-Host " "
-Write-Host "Starting Managed Folder Assistant..."
+Write-Host "`nStarting Managed Folder Assistant..."
```

Comment:
@ryanfkeepers I have https://github.com/alcionai/corso/pull/2818 to remove this last section. Maybe remove it here instead since PRs are taking so long to merge. 

It was failing on a different domain and it does not really work or is needed based on some additional research

Reply:
sounds good